src/commands/bundle.ts

```diff
@@ -0,0 +1,67 @@
+import Command from '../base';
+import { Flags } from '@oclif/core';
+import { load } from '../models/SpecificationFile';
+import { outputFlag } from '../flags';
+import path from 'path'; 
+import fs from 'fs';
+// @ts-ignore
+import yaml from 'js-yaml';
+// @ts-ignore
```

Comment:
```suggestion
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
```
One thing you can do is this ⏫ for now.